cmd/ocsp-responder/main.go

```diff
@@ -123,6 +104,10 @@ type Config struct {
 		// LogSampleRate sets how frequently error logs should be emitted. This
 		// avoids flooding the logs during outages. 1 out of N log lines will be emitted.
 		LogSampleRate int `validate:"min=0"`
+
+		// Deprecated: ExpectedFreshness is no longer used now that we do not read
+		// OCSP Response bytes from the database.
```

Comment:
Please open a TODO issue for removing this field and comment it.

Reply:
IN-8855